docker/local-development.md

```diff
@@ -9,17 +9,21 @@ the pattern to other projects. Some examples in the wild:
 
 ## Overview
 
+N.b., this documentation was written for containerized web applications
+written in a Python web framework (e.g., Django) and leveraging a Postgres
+database. YMMV!
```

Comment:
I don't think we actually need this warning, particularly as the first sentence in these docs. It seems clear from the snippets and templates that so far we're only documenting Django and Postgres (which is our standard stack anyway). I think we can save this until we have more templates to show, at which point we can help route the user to the correct template.

On the other hand, if you feel strongly that the warning is important to guide the reader, perhaps we could make this clearer in the title of the docs (e.g. `Docker for Local Django Development`).